test/functional/VM_Test/playlist.xml

```diff
@@ -174,4 +174,30 @@
 		</impls>
 	</test>
 
+	<test>
+		<testCaseName>j9vmClassUnloading</testCaseName>
+		<variations>
+			<variation>Mode110</variation>
+			<variation>Mode301</variation>
+			<variation>Mode351</variation>
+			<variation>Mode501</variation>
+			<variation>Mode551</variation>
+			<variation>Mode610</variation>
+		</variations>
+		<command>$(JAVA_COMMAND) -Xdump -Xint -Dj9vm.testing.tests.path=$(TEST_RESROOT) \
+	-cp $(Q)$(TEST_RESROOT)$(D)VM_Test.jar$(P)$(JVM_TEST_ROOT)$(D)VM$(D)lib$(D)asm-all.jar$(Q)  \
+	j9vm.runner.Menu -exe=$(SQ)$(TEST_JDK_HOME)$(D)bin$(D)java $(JVM_OPTIONS) -Xdump$(SQ) -jar=$(Q)$(TEST_RESROOT)$(D)VM_Test.jar$(Q) -test=$(Q)j9vm.test.classunloading.testcases$(Q) \
```

Comment:
$(TEST_JDK_HOME)$(D)bin$(D)java -> $(JAVA_COMMAND)

Reply:
$(JAVA_COMMAND) has its own quotation, which is not expected